test/TritonGPU/combine-select-if.mlir

```diff
@@ -1,46 +1,83 @@
-// RUN: triton-opt %s -split-input-file -tritongpu-combine-tensor-select-and-if | FileCheck %s
+// RUN: triton-opt %s -split-input-file --canonicalize -tritongpu-combine-tensor-select-and-if | FileCheck %s
```

Comment:
why do you need to run `--canonicalize`, that defeats the purpose of making this a unit test as it would run canonicalizer patterns